lib/game.js

```diff
@@ -68,43 +76,47 @@ function drawGround () {
   ceiling.move(groundDirection)
 }
 
+function drawBird (direction) {
+  bird.draw(context)
+  bird.fly(direction)
+}
+
 function checkForCollision () {
   obstacles.forEach(function(obstacle) {
     if (bird.x < obstacle.x + obstacle.width &&
       bird.x + bird.width > obstacle.x &&
       bird.y < obstacle.y + obstacle.height &&
       bird.height + bird.y > obstacle.y) {
+      saveHighScore()
+      // canvas.effect( "shake", 1000);
       location.reload()
+
     }
   })
 }
 
-context.font = "40px Arial";
-context.fillText("Press S to play", 175, 300, 300);
-
 function gameLoop () {
   context.clearRect(0, 0, canvas.width, canvas.height)
-  bird.draw(context)
-  bird.fly(direction)
-
-  drawGround();
+  drawBird(direction);
   drawObstacles();
+  drawGround();
   checkForCollision()
   updateScore();
-  saveHighScore();
   requestAnimationFrame(gameLoop)
 }
 
 window.addEventListener('keyup', function (event) {
-  if (event.keyCode === 32) { bird.flap() }
```

Comment:
stylistic/opinion thing here, but I generally prefer to inline super short if statements